client/PRUserial485/PRUserial485.py

```diff
@@ -161,101 +135,157 @@ def socket_communicate(conn_port, data_queue):
 
         # Receive data/payload
         payload = b''
-        if(data_size):
+        if data_size:
             try:
-                for i in range(int(data_size / 4096)):
+                for _ in range(int(data_size / 4096)):
                     payload += remote_socket.recv(4096, socket.MSG_WAITALL)
-                payload += remote_socket.recv(int(data_size % 4096), socket.MSG_WAITALL)
+                payload += remote_socket.recv(
+                    int(data_size % 4096), socket.MSG_WAITALL)
             except ConnectionResetError:
                 # This except might happen when server is suddenly stopped
                 payload = b''
 
         # Store answer and notify function
-        answers[report_event] = (command_recv, payload)
+        COMM_RESPONSE[report_event] = (command_recv, payload)
         report_event.set()
 
 
-
-def send_communication_data(payload):
+def _send_communication_data(payload):
+    """."""
     # Creates notification event
     notification_event = Event()
 
     # Add command into queue
-    if payload[0] == ord(_c.COMMAND_PRUserial485_write) or payload[0] == ord(_c.COMMAND_PRUserial485_read):
-        rw_queue.put([payload[0], payload, notification_event])
+    if payload[0] == ord(_c.COMMAND_PRUserial485_write) or \
+       payload[0] == ord(_c.COMMAND_PRUserial485_read):
+        QUEUE_RW.put([payload[0], payload, notification_event])
     else:
-        general_queue.put([payload[0], payload, notification_event])
+        QUEUE_GENERAL.put([payload[0], payload, notification_event])
 
     # Wait command complete. Get answer and delete event
     notification_event.wait()
     notification_event.clear()
-    command, payload_recv = answers.pop(notification_event, None)
+    command, payload_recv = COMM_RESPONSE.pop(notification_event, None)
     del notification_event
 
     return command, payload_recv
 
 
+# --- PRUserial485 API ---
+
+
+def PRUserial485_set_bbb_ip_address(ip_address):
+    """Define beaglebone IP address."""
+    global BBB_IP
+    BBB_IP = ip_address
```

Comment:
if "10.128." in ip_address and \
----len(ip_address.split(".")) == 4:
----BBB_IP = ip_address


Garantir que seja um IP válido e dentro da rede de controle

Reply:
done!